packages/client/components/MeetingLockedOverlay.tsx

```diff
@@ -124,8 +124,8 @@ const MeetingLockedOverlay = (props: Props) => {
   }, [locked])
 
   const onClick = () => {
-    SendClientSegmentEventMutation(atmosphere, 'Upgrade Intent', {
-      source: 'Direct Meeting Link Upgrade CTA',
+    SendClientSegmentEventMutation(atmosphere, 'Upgrade CTA Clicked', {
+      upgradeCTALocation: 'directMeetingLinkLock',
```

Comment:
Both the name and property changes throughout this PR are to line up with the events already in production for the template limits experiment.